src/main/resources/edu/cornell/kfs/kim/impl/identity/PersonImpl.xml

```diff
@@ -4,6 +4,14 @@
        xsi:schemaLocation="http://www.springframework.org/schema/beans
             http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
 
+    <bean id="PersonImpl" parent="PersonImpl-parentBean">
+        <property name="primaryKeys">
+            <list merge="true">
+                <value>principalName</value>
```

Comment:
If you don't need the getInquiryUrl() method change in the customized inquirable class, then you may not need to make this primary key change to the bean either.